MaterialDesignThemes.Wpf/TimePicker.cs

```diff
@@ -74,6 +76,8 @@ private static void SelectedTimePropertyChangedCallback(DependencyObject depende
 			timePicker.SetCurrentValue(TextProperty, timePicker.DateTimeToString(timePicker.SelectedTime));
             timePicker._isManuallyMutatingText = false;
             timePicker._lastValidTime = timePicker.SelectedTime;
+
+		    timePicker.SelectedTimeChanged?.Invoke(timePicker, new EventArgs());
```

Comment:
Personally I think it would be better to do this as a RoutedEvent with its own custom event args class that contains the new value. Very similar to how [TextBoxBase.TextChanged](https://msdn.microsoft.com/en-us/library/system.windows.controls.primitives.textboxbase.textchanged.aspx) works.

Reply:
I changed it to be a routed event. I originally made it a standard event by looking at the framework DatePicker, but sure, routed event works as well 👍 Sorry it took 7 months, wasn't easy you know 😁

Reply:
Great work, however, there was another PR that has already resolved this issue. Thank you for your willingness to contribute to this project!